pom.xml

```diff
@@ -739,6 +739,127 @@
+            <artifactId>jsch</artifactId>
+            <version>0.1.54</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.httpcomponents</groupId>
+            <artifactId>httpclient</artifactId>
+            <version>4.5.3</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.httpcomponents</groupId>
+            <artifactId>httpcore</artifactId>
+            <version>4.4.6</version>
+        </dependency>
+        <dependency>
+            <groupId>org.objenesis</groupId>
+            <artifactId>objenesis</artifactId>
+            <version>2.4</version>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-beans</artifactId>
+            <version>2.5.6.SEC03</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.ant</groupId>
+            <artifactId>ant</artifactId>
+            <version>1.9.2</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.maven</groupId>
+            <artifactId>maven-embedder</artifactId>
+            <version>3.5.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.eclipse.sisu</groupId>
+            <artifactId>org.eclipse.sisu.plexus</artifactId>
+            <version>0.3.3</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.maven</groupId>
+            <artifactId>maven-core</artifactId>
+            <version>3.5.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.maven</groupId>
+            <artifactId>maven-aether-provider</artifactId>
+            <version>3.3.1</version>
+        </dependency>
+        <dependency>
+            <groupId>commons-io</groupId>
+            <artifactId>commons-io</artifactId>
+            <version>2.5</version>
+        </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins.icon-shim</groupId>
+            <artifactId>icon-set</artifactId>
+            <version>2.0.3</version>
+        </dependency>
+        <dependency>
+            <groupId>org.codehaus.plexus</groupId>
+            <artifactId>plexus-classworlds</artifactId>
+            <version>2.5.2</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.maven</groupId>
+            <artifactId>maven-compat</artifactId>
+            <version>3.5.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.ow2.asm</groupId>
+            <artifactId>asm</artifactId>
+            <version>5.0.3</version>
+        </dependency>
+        <dependency>
+            <groupId>org.ow2.asm</groupId>
+            <artifactId>asm-tree</artifactId>
+            <version>5.0.3</version>
+        </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci</groupId>
+            <artifactId>symbol-annotation</artifactId>
```

Comment:
No, you should never add this dep directly. Depend on `structs` where you actually need `@Symbol`.

Reply:
ok.

Reply:
@jglick github-branch-source has (transitive) dependency on symbol-annotation 1.5 where as core and struts have direct dependency on ver 1.1. Can't depend on struts for this. Either we define dependency on it directly or somehow exclude it from core and struts?

```
Require upper bound dependencies error for org.jenkins-ci:symbol-annotation:1.1 paths to dependency are:
+-io.jenkins.blueocean:blueocean-bitbucket-pipeline:1.3.0-beta-3-SNAPSHOT
  +-org.jenkins-ci.main:jenkins-core:2.46.3
    +-org.jenkins-ci:symbol-annotation:1.1
and
+-io.jenkins.blueocean:blueocean-bitbucket-pipeline:1.3.0-beta-3-SNAPSHOT
  +-org.jenkins-ci.plugins:cloudbees-bitbucket-branch-source:2.2.3
    +-org.jenkins-ci.plugins:structs:1.10
      +-org.jenkins-ci:symbol-annotation:1.1
and
+-io.jenkins.blueocean:blueocean-bitbucket-pipeline:1.3.0-beta-3-SNAPSHOT
  +-io.jenkins.blueocean:blueocean-pipeline-api-impl:1.3.0-beta-3-SNAPSHOT
    +-org.jenkins-ci.plugins:github-branch-source:2.2.3
      +-com.coravy.hudson.plugins.github:github:1.28.0 (managed) <-- com.coravy.hudson.plugins.github:github:1.26.0
        +-org.jenkins-ci:symbol-annotation:1.5
```

Reply:
This is a bug in `github-plugin/pom.xml` which I will file a PR for. You must never add a dep on `symbol-annotation` from a plugin. Depend on `structs`, or exclude this dep from `github` for now.

Reply:
https://github.com/jenkinsci/github-plugin/pull/176